docs/content/preview/drivers-orms/java/yugabyte-jdbc.md

```diff
@@ -188,7 +211,8 @@ If you receive no output or an error, check the parameters in the connection str
 
 ## Learn more
 
-- Build Java applications using [Hibernate ORM](../hibernate/)
-- [YugabyteDB JDBC driver reference](../../../reference/drivers/java/yugabyte-jdbc-reference/#load-balancing)
+- Refer to [YugabyteDB JDBC driver reference](../../../reference/drivers/java/yugabyte-jdbc-reference/#load-balancing) and [Try it out](../../../reference/drivers/java/yugabyte-jdbc-reference/#try-it-out) for detailed smart driver examples.
```

Comment:
Don't deeplink the first link, I may just pull the load balance section now we have smart driver page